src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -368,6 +433,14 @@ public static BigIntUnsignedHandlingMode parse(String value, String defaultValue
             .withValidation(CommonConnectorConfig::validateTopicName)
             .withDescription("Overrides the topic.prefix used for the data change topic.");
 
+    public static final Field OVERRIDE_SCHEMA_CHANGE_TOPIC = Field.create("override.schema.change.topic")
```

Comment:
Please remove this one, it can be solved using https://debezium.io/documentation/reference/3.0/connectors/vitess.html#vitess-property-topic-naming-strategy